plugins/source_scheduler_dfs.go

```diff
@@ -74,15 +74,24 @@ func (p *SourcePlugin) syncDfs(ctx context.Context, spec specs.Source, client sc
 	wg.Wait()
 }
 
+func (p *SourcePlugin) logTablesMetrics(tables schema.Tables, client schema.ClientMeta) {
+	clientName := client.ID()
+	for _, table := range tables {
+		metrics := p.metrics.TableClient[table.Name][clientName]
+		p.logger.Info().Str("table", table.Name).Str("client", clientName).Uint64("resources", metrics.Resources).Uint64("errors", metrics.Errors).Msg("table sync finished")
+		p.logTablesMetrics(table.Relations, client)
+	}
+}
+
 func (p *SourcePlugin) resolveTableDfs(ctx context.Context, table *schema.Table, client schema.ClientMeta, parent *schema.Resource, resolvedResources chan<- *schema.Resource, depth int) {
 	clientName := client.ID()
 	logger := p.logger.With().Str("table", table.Name).Str("client", clientName).Logger()
 
 	if parent == nil { // Log only for root tables, otherwise we spam too much.
-		logger.Info().Msg("table resolver started")
+		logger.Info().Msg("root table resolver started")
```

Comment:
```suggestion
		logger.Info().Msg("top level table resolver started")
```

I think this is how we refer to these tables in other places